src/transform.ts

```diff
@@ -103,6 +100,20 @@ export interface AggregatedFieldDef {
   as: string;
 }
 
+
+export type WindowOnlyOp =
+  |'row_number'
```

Comment:
Why starting with |

Reply:
I was following the example for aggregate op in transform.d.ts, to make the style more consistent.

```typescript
export type AggregateOp =
  | 'argmax'
  | 'argmin'
  | 'average'
  | 'count'
  | 'distinct'
  | 'max'
  | 'mean'
  | 'median'
  | 'min'
  | 'missing'
  | 'q1'
  | 'q3'
  | 'ci0'
  | 'ci1'
  | 'stderr'
  | 'stdev'
  | 'stdevp'
  | 'sum'
  | 'valid'
  | 'values'
  | 'variance'
  | 'variancep';
```

Reply:
Interesting.   I don't know what's the rationale for `transform.d.ts` though. 
I'd argue that this is a bad style, so we shouldn't follow it.

Reply:
That's the style in vega-typings. It might be automatically generated by prettier over there. You can ignore the style for Vega-Lite.